tests/runtests.py

```diff
@@ -518,6 +518,21 @@ def test_cmd_highlight(self):
         self.assertIn("\x1b[35m44444444\x1b[0m", res)
         return
 
+    def test_cmd_aliases(self):
+        # test add functionality
+        add_res = gdb_start_silent_cmd("aliases add alias_function_test example")
+        self.assertNoException(add_res)
+        # test list functionality
+        list_res = gdb_start_silent_cmd("aliases ls")
+        self.assertNoException(list_res)
+        self.assertIn("alias_function_test", list_res)
```

Comment:
This CI test is failing 
```
'alias_function_test' not found
```

That's because every `gdb_{run,start}_...*` runs in its own temporary environment, isolated from the rest. Because we don't save, `alias_function_test` doesn't exist after L523. Use the `before=` kwarg for that 

```python
list_res = gdb_start_silent_cmd("aliases ls", before="aliases add alias_function_test example")
[...]
self.assertIn("alias_function_test", list_res)
```




See https://github.com/hugsy/gef/runs/2769822919?check_suite_focus=true